zap/src/main/java/org/zaproxy/zap/model/TechSet.java

```diff
@@ -117,4 +119,26 @@ public void print() {
             System.out.println("\tExclude: " + tech);
         }
     }
+
+    public static TechSet getAllTech() {
```

Comment:
Should have JavaDoc explaining that this returns a new `TechSet` each time, although it should not need to.